config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -136,8 +134,6 @@ def build_test(cloud='aws',
         env['CLUSTER_NAME'] = f"e2e-{name_hash[0:10]}-{name_hash[12:17]}.test-cncf-aws.k8s.io"
         env['KOPS_STATE_STORE'] = 's3://k8s-kops-prow'
         env['KUBE_SSH_USER'] = kops_ssh_user
-        if irsa and cloud == "aws":
-            env['KOPS_IRSA'] = "true"
```

Comment:
Scenarios ignore `extra_flags`, so `KOPS_IRSA` is needed.

Reply:
But, is there anything using it on kOps side?

Reply:
https://github.com/kubernetes/kops/blob/6845a8e648ac76a7ed84bf3f2eac5f17a582a03e/tests/e2e/scenarios/lib/common.sh#L49-L53

Reply:
Thanks, somehow I missed it. Reverting.